python/lsst/ts/standardscripts/auxtel/standby_atcs.py

```diff
@@ -0,0 +1,55 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["StandbyATCS"]
+
+from ..standby_group import StandbyGroup
+from lsst.ts.observatory.control.auxtel.atcs import ATCS, ATCSUsages
+
+
+class StandbyATCS(StandbyGroup):
+    """Put ATCS components in standby.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
```

Comment:
Please add "None" (or similar) as the contents of this section.

Otherwise users may think you forgot to fill in this section.